packages/icons/icons.json

```
@@ -3726,84 +3726,84 @@
     {
         "displayName": "Waves",
         "iconName": "waves",
-        "tags": "sea, ocean, tide",
+        "tags": "sea, ocean, tide, grass, water",
         "group": "miscellaneous",
         "codepoint": 62231
     },
     {
         "displayName": "Rain",
         "iconName": "rain",
-        "tags": "precipitation, storm, weather",
+        "tags": "precipitation, storm, weather, cloud, data",
         "group": "miscellaneous",
         "codepoint": 62085
     },
     {
         "displayName": "Lightning",
         "iconName": "lightning",
-        "tags": "thunder, bolt, strike, weather",
+        "tags": "thunder, bolt, strike, weather, quick, flash, immediate, electricity, power, high voltage, spark",
         "group": "miscellaneous",
         "codepoint": 61996
     },
     {
         "displayName": "Wind",
         "iconName": "wind",
-        "tags": "air, breath, draft, weather",
+        "tags": "air, breath, draft, weather, blowing, current",
         "group": "miscellaneous",
         "codepoint": 62236
     },
     {
         "displayName": "Hurricane",
         "iconName": "hurricane",
-        "tags": "storm, cyclone, typhoon, tropical storm",
+        "tags": "storm, cyclone, typhoon, tropical storm, swirl, fast, razor, saw, spin, whirl pool",
         "group": "miscellaneous",
         "codepoint": 61942
     },
     {
         "displayName": "Nest",
         "iconName": "nest",
-        "tags": "nest, indent, down right",
+        "tags": "indent, down right, folder, click, next, forward, continue",
         "group": "interface",
         "codepoint": 62032
     },
     {
         "displayName": "Target",
         "iconName": "target",
-        "tags": "location, pinpoint, scope, center",
+        "tags": "location, pinpoint, scope, center, add, new, plus",
```

the "add" icon is much better suited for "add, new, plus"

```suggestion
        "tags": "location, pinpoint, scope, center",
```